Database.php

```diff
@@ -28,8 +28,8 @@
 	'hostsByName' => [
 		// 81.4.109.166
 		'db4' => 'mediawiki-internal-db-master.miraheze.org',
-		// 168.235.110.50
-//		'db5' => 'mediawiki-internal-db-master-db5.miraheze.org',
+		// 185.52.1.89
+		'db5' => 'mediawiki-internal-db-master-db5.miraheze.org',
 	],
 	'externalLoads' => [
```

Comment:
Shouldn't db5 be uncommented as well? (not sure!)

Reply:
In externalLoads no, because that means it'll balance between db4 and db5 (meaning a replica i think).